src/modules/common/helpers/deep.clone.helper.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Clones any object. Faster alternative to `JSON.parse(JSON.stringify)`
+ *
+ * @function deepClone
+ * @param {Any} data
+ * @returns {Any} cloned data
+ */
+const deepClone = (data) => {
```

Comment:
Did you make some speed tests? I have made few simple tests and it seems that the speeds are basicaly the same.
But these were quite simple speed tests. :)

Reply:
My simple test suite: https://jsbin.com/zetanapoha/edit?js,output 🏎

Reply:
This is the same approach we're using in Page Builder. I wanted to make it inline.

Reply:
> Inline
I don't know what you mean by that?

Reply:
It is just to make it behave the same way as in Page Builder?

Reply:
Exactly

Reply:
I think that deep clone output should always be the same, why not use previous approach from UDW in Page Builder? It is much simpler and it seems that performance is the same. But nevermind. :)

Reply:
I did other performance tests and it told me that the previous approach was faster only in FF. In other browsers this approach was better.
We can discuss it in private and, eventually, prepare test cases for each option :)

Reply:
If you performed more detailed tests and your approach performed better then OK. :)
Maybe in the future will find some time to test it more thoroughly. :P